components/brave_ads/core/public/prefs/pref_names.h

```diff
@@ -47,6 +47,8 @@ inline constexpr char kSaveAds[] = "brave.brave_ads.reactions.saved_ads";
 inline constexpr char kMarkedAsInappropriate[] =
     "brave.brave_ads.reactions.marked_as_inappropriate";
 
+inline constexpr char kHasMigratedState[] =
```

Comment:
Maybe we should we indicate in name that we migrate to v2 state? 
Just in case if we have some other `state` migration in future.

Reply:
This is hasMigratedV1, so future functions would have v2 etc.